src/App.tsx

```diff
@@ -14,11 +16,78 @@ const goodsFromServer: string[] = [
   'Garlic',
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+type State = {
+  selectedGood: string;
+};
+
+class App extends Component {
```

Comment:
add types to Class
https://mate-academy.github.io/fe-program/js/extra/react-typescript#class-component